client/css/ProductOverview.scss

```diff
@@ -47,33 +64,51 @@
 .socialMedia {
   display: flex;
   margin: 10px 0;
-  justify-content: left;
-}
+  justify-content: space-between;
+  width: 50%;
 
-.styleIndicator {
-  font-weight: bold;
-}
+  svg {
+    transition: color 200ms;
```

Comment:
I think this transition is redundant as you already state the transition property in the onHover case.

Reply:
You need it going in both directions. transition in as well as transition back. Otherwise, onHover would be smooth and offHover would be abrupt. Play with it and see.